lib/route/routes.dart

```diff
@@ -1,19 +1,33 @@
-import 'package:admin_dashboard/page/login_page.dart';
+import 'package:admin_dashboard/dto/admin_dashboard_cache_model.dart';
+import 'package:admin_dashboard/page/repo_page.dart';
+import 'package:admin_dashboard/page/welcome_page.dart';
+import 'package:admin_dashboard/service/fire_base.dart';
 import 'package:flutter/material.dart';
 
 ///The route generator that handles navigation throughout the app\
 
 class RouteGenerator {
   RouteGenerator._();
-  ///login page path
-  static const String loginPage = '/';
+  ///Welcome page path
+  static const String welcomePage = '/';
+  ///repository page path
+  static const String repoPage = '/repos';
+
+
 
   ///decides which page to open depending on the path provided to the navigator
-  static Route<dynamic> generateRoute(RouteSettings settings) {
+  static Route<dynamic> generateRoute(RouteSettings settings,
+      AdminDashboardCache cache,
+      FireBaseService fire,
+      ) {
     switch (settings.name) {
-      case loginPage:
+      case welcomePage:
+        return MaterialPageRoute(
+          builder: (_) =>  WelcomePage(cache: cache, fire: fire),
+        );
+      case repoPage:
         return MaterialPageRoute(
-          builder: (_) => const LoginPage(),
+            builder: (_) => RepoPage(cache: cache, fire: fire),
```

Comment:
Let's change this to GoRouter package but not in this PR, maybe later. we would need GoRouter nested URL for web later.